node-backend/app/controllers/v1/profileController.js

```diff
@@ -0,0 +1,21 @@
+const db = require("../../models/sequelize");
+const Users = db.users;
+
+const jwtToken = require("../../jwtTokenValidation/jwtValidation");
+module.exports.getProfile = async (req, res) => {
+  const authHeader = req.headers["authorization"];
+  let userData = await jwtToken.getData(authHeader);
```

Comment:
you can directly pass req.headers["authorization"] to function no need to get value in variable